pkg/dev/dev.go

```diff
@@ -2,20 +2,24 @@ package dev
 
 import (
 	"context"
+	"fmt"
 	"io"
 
 	"github.com/redhat-developer/odo/pkg/binding"
 	"github.com/redhat-developer/odo/pkg/envinfo"
 	"github.com/redhat-developer/odo/pkg/kclient"
 	"github.com/redhat-developer/odo/pkg/portForward"
 	"github.com/redhat-developer/odo/pkg/preference"
-	"github.com/redhat-developer/odo/pkg/testingutil/filesystem"
+	filesystem "github.com/redhat-developer/odo/pkg/testingutil/filesystem"
```

Comment:
nit: is this import alias needed? I find it a bit redundant.
https://github.com/redhat-developer/odo/wiki/Dev:-Coding-Conventions#import-aliasing